rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -23,6 +23,9 @@ elasticsearch_legacy_apt_repos:
 # use -e 'logging_upgrade=true' to upgrade logging components
 logging_upgrade: false
```

Comment:
@git-harry I have that set to false so that we can re-use the plays for a fresh elasticsearch installation.  I suppose we could add some checks for legacy indices in the actuall `elasticsearch_pre_install.yml` to ensure that the upgrade does not happen unintentionally.